src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: #060b35;
+    font-size: 12px;
+    line-height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &:hover {
+      color: var(--primary-color);
+      text-decoration: none;
+    }
+  }
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+
+  &::after {
+    content: '';
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    background-color: var(--primary-color);
+    position: absolute;
+    bottom: 0;
+    left: 0;
```

Comment:
The `left: 0;` property is not necessary if the width of the pseudo-element is 100%. It will automatically start from the left edge of the container.